packages/blade-svelte/src/components/Button/Button.stories.svelte

```diff
@@ -0,0 +1,103 @@
+<script context="module">
```

Comment:
button's color looks a bit too light in darkmode: 

<img width="137" height="66" alt="Image" src="https://github.com/user-attachments/assets/c1805635-6777-403b-bf9c-f38097cc0494" />